python/social_bot/goal_task.py

```diff
@@ -27,19 +39,22 @@ def __init__(self,
                  max_steps=500,
                  goal_name="goal",
                  success_distance_thresh=0.5,
-                 fail_distance_thresh=0.5):
+                 fail_distance_thresh=0.5,
+                 goal_random_range=10.0):
```

Comment:
This change the original behavior, which is goal_random_range=2.0. To keep SimpleNavigation unaffected, you need to change the code there. A better way is to use a default which is same as the original behavior, so the existing code won't need to be changed.

Reply:
Yes, original behavior should not be changed. Fixed now.